File: demo-web-java/src/test/java/com/example/demo/repository/UserRepositoryTest.java
package com.example.demowebjava.repository;

import com.example.demowebjava.DemoWebJavaApplicationTests;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

import static org.junit.Assert.*;

public class UserRepositoryTest extends DemoWebJavaApplicationTests {

  @Autowired
  UserRepository userRepository;

  @Test
  public void findByEmail() throws Exception {
    assertNotNull(userRepository.findByEmail("admin@rsupport.com"));
  }

}

File: demo-web-scala/src/test/scala/com/example/demo/repository/UserRepositoryTest.scala
package com.example.demo.repository

import org.junit.Test
import org.springframework.beans.factory.annotation.Autowired

class UserRepositoryTest extends AbstractTest {

  @Autowired
  var userRepository:UserRepository = null

  @Test
  def findActiveUsers() = {
   val list = userRepository.findActiveUsers()
  log.debug("{}", list)
  }

}
